### ISSUE TITLE
Don't work on macos mojave

### ISSUE THREAD

**Issue by shirshak55** [TablePlus/DBngin#8]
Please fill out the detail below, it helps me investigate the bug:

1. Driver (Ex: PostgreSQL 10.0):
Mysql 8.0.12

2. DBngin build number:

3. macOS version:
Majove

4. The steps to reproduce this issue:
Just download and try to install laravel and run php artisan migrate it don't work properly

Not just laravel PDO connection is not working too.

But table plus work that is confusing me

**Comment by shirshak55** [comment]
ok it woks on 5.7 but v8 has error . Can u update to latest mysql

**Comment by huyphams** [comment]
Yup, thanks. Let's me see what is the error because I'm using Mojave (upgrade) and it works fine.

**Comment by shirshak55** [comment]
@huyphams were you able to use Mysql 8.0.12 using other than table plus? Laravel etc don't seem to work

**Comment by huyphams** [comment]
MySQL 8.0.12 requires the newest driver from Oracle, I don't know fi Laravel has upgraded their lib or not. If you check the error and it says `caching_sha2_password plugin is missing` it's simply because Laravel doesn't support it.

**Comment by shirshak55** [comment]
@huyphams  hi from where did you check error?

**Comment by mkwsra** [comment]
![image](https://user-images.githubusercontent.com/1743919/108708350-63490600-7522-11eb-90e0-92e7d6ad9078.png)

I have the same issue
